CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 # Changelog
 
-## Unreleased
+## 5.23.0-alpha.1
+
+### Fixes
+
+- Pass `replaysSessionSampleRate` option to Android ([#3714](https://github.com/getsentry/sentry-react-native/pull/3714))
+
+Access to Mobile Replay is limited to early access orgs on Sentry. If you're interested, [sign up for the waitlist](https://sentry.io/lp/mobile-replay-beta/)
```

Comment:
Shouldn't Replay be set on `### Features` ?

```suggestion

### Features

- Access to Mobile Replay is limited to early access orgs on Sentry. If you're interested, [sign up for the waitlist](https://sentry.io/lp/mobile-replay-beta/)

### Fixes

- Pass `replaysSessionSampleRate` option to Android ([#3714](https://github.com/getsentry/sentry-react-native/pull/3714))

```

Reply:
Yes, absolutely and it was in the first alpha
https://github.com/getsentry/sentry-react-native/pull/3714/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR80

But since the second is building on to of it I put it in as fix.

For the stable release we will repeat the ## Features section.